### PR TITLE
Added http example

### PR DESCRIPTION
**Pull request by MichaelS11**
Added Duration to time package

### CODE REVIEW

**Comment by MichaelS11** [comment]
@mattn 

This is the code I would like to use for the example:

`client = &http.Client{Timeout: 30 * time.Second}`

Which doesn't work.

So tried to get this code to work:

```
duration = make([]time.Duration)
duration[0] = 30 * time.Second
client = http.DefaultClient
client.Timeout = duration[0]
```
Which gives the error: "execute error: type time.Duration cannot be assigned to type time.Duration for struct"

The main issue is if there is a way to make Anko be able to assign values to a struct?

**Comment by MichaelS11** [comment]
@mattn 

Any thoughts about the struct assign issue above?

File: packages/time.go [hunk 1]
```diff
@@ -28,8 +29,10 @@ func init() {
 		"Minute":          time.Minute,
 		"Hour":            time.Hour,
 	}
+	var duration time.Duration
 	PackageTypes["time"] = map[string]interface{}{
-		"Ticker": time.Ticker{},
-		"Time":   time.Time{},
+		"Duration": reflect.TypeOf(&duration).Elem(),
```
Comment (mattn) [inline packages/time.go]: time.Duration(0) ?

Comment (MichaelS11) [reply packages/time.go]: Updated

**Comment by mattn** [comment]
Thank you

**Pull request merged**